src/pages/App.tsx

```diff
@@ -75,7 +76,11 @@ const App = () => (
                               path={Path.VOTE_LEADER_BOARD}
                               component={VoterLeaderboard}
                             />
-                            <Route exact path={Path.VOTE_PROPOSAL} component={VoteOverview} />
+                            <Route
+                              exact
+                              path={Path.VOTE_PROPOSAL}
```

Comment:
To keep the same pattern as for market/market details, I'd name this `VOTE_PROPOSAL_DETAILS`

Reply:
done